ProcessLib/LiquidFlow/LiquidFlowMaterialProperties.h

```diff
@@ -0,0 +1,96 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   LiquidFlowMaterialProperties.h
+ *
+ * Created on August 18, 2016, 11:03 AM
+ */
+
+#ifndef OGS_LIQUIDFLOWMATERIALPROPERTIES_H
+#define OGS_LIQUIDFLOWMATERIALPROPERTIES_H
+
+#include <memory>
+
+#include "MaterialLib/Fluid/FluidPropertyHeaders.h"
+#include "MaterialLib/PorousMedium/PorousPropertyHeaders.h"
+
+#include "MeshLib/PropertyVector.h"
+
+namespace MeshLib
+{
+class Mesh;
+}
+
+namespace MaterialLib
+{
+namespace PorousMedium
+{
+class Porosity;
+class Storage;
+}
+}
+
+namespace ProcessLib
+{
+namespace LiquidFlow
+{
+struct LiquidFlowMaterialProperties
+{
+    typedef MaterialLib::Fluid::FluidProperty::ArrayType ArrayType;
+
+    explicit LiquidFlowMaterialProperties(BaseLib::ConfigTree const& config);
```

Comment:
Usually a create function is used for better testability of the class and to separate parsing from actual construction.